reconcile/utils/three_way_diff_strategy.py

```diff
@@ -152,8 +147,7 @@ def three_way_diff_using_hash(c_item: OR, d_item: OR) -> bool:
         item for item in patch.patch if is_valid_change(current, desired, item)
     ]
     if len(valid_changes) > 0:
-        logging.info("Desired and Current objects differ -> Apply")
-        logging.info(valid_changes)
+        logging.debug(f"Desired and Current objects differ -> Apply: {valid_changes}")
```

Comment:
for logging method, best to use const string:

```
logging.debug("Desired and Current objects differ -> Apply: %s", valid_changes)
```

see https://docs.python.org/3/howto/logging.html#optimization and https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/logging-format-interpolation.html